### PR TITLE
Issues in public

### PR DESCRIPTION
**Pull request by MSilva95**
fix(ascii-art): fixing the way to use the flag on the optional projects

### CODE REVIEW

File: subjects/ascii-art/output/audit.md [hunk 1]
```diff
@@ -2,17 +2,21 @@
 
 ###### Has the requirement for the allowed packages been respected? (Reminder for this project: only [standard packages](https://golang.org/pkg/))
 
+<<<<<<< Updated upstream
```
Comment (nprimo) [inline subjects/ascii-art/output/audit.md:5]: an auto generated line from git to be removed

File: subjects/ascii-art/output/audit.md [hunk 2]
```diff
@@ -2,17 +2,21 @@
...
 ##### Try passing as arguments `"banana" standard --output test00.txt`
+=======
```
[the remark this answers is not in the document]
Comment (MSilva95) [reply subjects/ascii-art/output/audit.md]: fixed

File: subjects/ascii-art/output/audit.md (hunk 1 above)
Comment (MSilva95) [reply subjects/ascii-art/output/audit.md]: fixed

**Review by nprimo — APPROVED** [review]
Everything looks good 🚀

**Pull request merged**